test-fixtures/src/main/java/org/sonar/api/testfixtures/log/LogAndArguments.java

```diff
@@ -19,37 +19,28 @@
  */
 package org.sonar.api.testfixtures.log;
 
+import ch.qos.logback.classic.spi.IThrowableProxy;
 import java.util.Arrays;
 import java.util.Optional;
+import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
 public final class LogAndArguments {
   private final String rawMsg;
+  private final IThrowableProxy throwable;
```

Comment:
this exposes a logback class, but I think it's ok since this is just a testing util, and this way we have access to the full exception stack if needed, without having to rewrite our own classes for it.

Reply:
I agree we should hide logback-specific inside LogTester as much as possible, so @damien-urruty-sonarsource 's idea LGTM